test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs

```diff
@@ -57,9 +64,89 @@ public void W3CTraceContextTestSuite(string value)
                 string lastLine = ParseLastLine(result);
                 this.output.WriteLine("result:" + result);
                 Assert.StartsWith("FAILED", lastLine);
+#if NETCOREAPP2_1
+                AssertOutputOfNetCoreApp21(result);
+#else
+                AssertOutputOfNetCoreApp31(result);
+#endif
+            }
+        }
+
+        private static void AssertOutputOfNetCoreApp21(string output)
```

Comment:
is the integration test CI running for both 2.1 and 3.1?

Reply:
Currently it only runs on 3.1. Created an issue to track making it run on 2.1 along with a couple other docker tests: #1264

Reply:
The issue has been resolved. Marking this comment as resolved now.